### PR TITLE
[Konvoy] documentation for SAML connector in dex

### PR DESCRIPTION
**Pull request by GoelDeepak**
## Jira Ticket
https://jira.mesosphere.com/browse/DCOS-61207

## Description of changes being made

Documents the steps to connect Konvoy to an Idp using SAML connector

Note: I have tested these steps using SSOCircle https://www.ssocircle.com/en/idp-tips-tricks/public-idp-configuration

### CODE REVIEW

File: pages/ksphere/konvoy/latest/security/external-idps/howto-dex-saml-connector/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+menuWeight: 10
+excerpt: Connect your Konvoy cluster to an Idp using SAML
+enterprise: false
+---
+
+## How to connect Konvoy to an Idp using SAML
+
+This guide shows how to configure your Konvoy cluster to use SAML to connect to an Idp.
+
+Step 1: Create a Konvoy cluster
```
Comment (cmastr) [inline pages/ksphere/konvoy/latest/security/external-idps/howto-dex-saml-connector/index.md]: Add a link to the quickstart section so they have info on how to add a cluster.

**Comment by GoelDeepak** [comment]
@jieyu  I didn't know we have a parallel docs in Konvoy. Thanks for pointing that out.

@cmastr Should I close this PR and open on under Konvoy? Or should I create another PR in Konvoy?

**Comment by cmastr** [comment]
> @jieyu I didn't know we have a parallel docs in Konvoy. Thanks for pointing that out.
> 
> @cmastr Should I close this PR and open one under Konvoy? Or should I create another PR in Konvoy?

Yeah Konvoy docs are done on the Konvoy repo. Up to you guys how you want to do this. We can work there and push to docs. Or work here and push up to the original repo.

**Comment by samvantran** [comment]
Should we consolidate docs into one central repo (here?). Maintaining two sets of docs will likely end up with one being outdated from the other.

**Comment by GoelDeepak** [comment]
I guess the workflow is to create docs in konvoy repo and let docs team push them to doc site.

**Comment by GoelDeepak** [comment]
I have created another PR in Konvoy repo https://github.com/mesosphere/konvoy/pull/1113

Should I close this one?

**Comment by cmastr** [comment]
> Should we consolidate docs into one central repo (here?). Maintaining two sets of docs will likely end up with one being outdated from the other.

The Konvoy team previously asked that we do docs on their repo. I'm fine with that, but don't know their reasons. The guidelines we had is that Doc has to have full access to 

> I have created another PR in Konvoy repo [mesosphere/konvoy#1113](https://github.com/mesosphere/konvoy/pull/1113)
> 
> Should I close this one?

my changes didn't get ported over to your new PR. Leave this one open until the other is approved. Then we can close without merging.

**Pull request closed without merging**